controllers/admin/AdminImportController.php

```diff
@@ -2049,24 +2708,27 @@ protected function productImportOne($info, $default_language_id, $id_lang, $forc
                 if (!is_array($product->tags) && !empty($product->tags)) {
                     $product->tags = AdminImportController::createMultiLangField($product->tags);
                     foreach ($product->tags as $key => $tags) {
-                        $is_tag_added = Tag::addTags($key, $product->id, $tags, $this->multiple_value_separator);
-                        if (!$is_tag_added) {
-                            $this->addProductWarning(Tools::safeOutput($info['name']), $product->id, $this->trans('Tags list is invalid', array(), 'Admin.Advparameters.Notification'));
+                        $isTagAdded = Tag::addTags($key, $product->id, $tags, $this->multiple_value_separator);
+                        if (!$isTagAdded) {
+                            $this->addProductWarning(
+                                Tools::safeOutput($productData['name']),
+                                $product->id,
+                                $this->trans('Tags list is invalid', array(), 'Admin.Advparameters.Notification')
+                            );
                             break;
                         }
                     }
                 } else {
                     foreach ($product->tags as $key => $tags) {
                         $str = '';
-                        foreach ($tags as $one_tag) {
-                            $str .= $one_tag.$this->multiple_value_separator;
+                        foreach ($tags as $oneTag) {
+                            $str .= $oneTag.$this->multiple_value_separator;
                         }
                         $str = rtrim($str, $this->multiple_value_separator);
 
-                        $is_tag_added = Tag::addTags($key, $product->id, $str, $this->multiple_value_separator);
-                        if (!$is_tag_added) {
-                            $this->addProductWarning(Tools::safeOutput($info['name']), (int)$product->id, $this->trans(
-                                'Invalid tag(s) (%s)',
+                        $isTagAdded = Tag::addTags($key, $product->id, $str, $this->multiple_value_separator);
+                        if (!$isTagAdded) {
+                            $this->addProductWarning(Tools::safeOutput($productData['name']), (int)$product->id, $this->trans('Invalid tag(s) (%s)',
```

Comment:
Same comment about translation placeholder. Please use a variable name surrounded by `%%` an add it in the second parameter.